pytorch/serve
# Adding Prometheus metrics framework (#460) (#602)

Merged commits from release/0.2.0 to master
* Test Prometheus client with sample metrics

* Updated documentation and additional labels for model metrics

* Updated inference unit tests to check for metrics; Added openapi docs

* Addressed review comments

* Added POSTMAN regression tests for metrics

* Update connecter types

* Bumping serving-sdk dependency

* Dropping metrics plugins support for now

* Fixed erroneous test cases

* added flag to enable or disable prometheous metric api

* Adding missing file

* doc changes and fixed typo

* Update configuration.md

made section for new params related to metric

* Update metrics_api.md

* Update MetricAggregator.java

Fixed constant name

Co-authored-by: dhaniram-kshirsagar <26479924+dhaniram-kshirsagar@users.noreply.github.com>
Co-authored-by: dhaniram kshirsagar <dhaniram_kshirsagar@persistent.com>

Co-authored-by: dhaniram-kshirsagar <26479924+dhaniram-kshirsagar@users.noreply.github.com>
Co-authored-by: dhaniram kshirsagar <dhaniram_kshirsagar@persistent.com>
